share/zmtest

```diff
@@ -12,9 +12,10 @@ usage () {
     echo "Usage: zmtest [OPTIONS] DOMAIN" >&2
     echo >&2
     echo "Options:" >&2
-    echo "  -s URL --server=URL   Zonemaster Backend to query. Default is http://localhost:5000/" >&2
+    echo "  -s URL --server URL   Zonemaster Backend to query. Default is http://localhost:5000/" >&2
     echo "  --noipv4              Run the test without ipv4." >&2
     echo "  --noipv6              Run the test without ipv6." >&2
+    echo "  --lang LANG           A language tag. Default is en." >&2
```

Comment:
I suggest:

```
A language tag. Default is "en".
```

Plus a reference to `backend-config.ini` and its documentation for valid language tags.

Reply:
I don't see how anyone would be confused by the lack of quotes. But if you insist I'll add them?

I'll add a note about valid parameter values.

Reply:
I think the quotes makes it clearer. And a note about valid parameters would be great. Thanks.